analyzers/src/SonarAnalyzer.CFG/Extensions/IsPatternExpressionSyntaxWrapperExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
```

Comment:
I just realised these extensions are added to SonarAnalyzer.CFG. Shouldn`t they be in the SonarAnalyzer.CSharp/Extensions?

Reply:
I want them to stay together with the LightUp layer. They are useful this layer is used (in this way we can reuse them in security).